pkg/util/runtime/features.go

```diff
@@ -43,18 +43,27 @@ const (
 
 	// NodeExternalDNS is a feature flag to enable/disable node ExternalDNS and InternalDNS use as GameServer address
 	NodeExternalDNS Feature = "NodeExternalDNS"
+
+	// FeatureStateAllocationFilter is a feature flag that enables state filtering on Allocation.
+	FeatureStateAllocationFilter Feature = "StateAllocationFilter"
+
+	// FeaturePlayerAllocationFilter is a feature flag that enables the ability for Allocations to filter based on
+	// player capacity.
+	FeaturePlayerAllocationFilter Feature = "PlayerAllocationFilter"
 )
 
 var (
 	// featureDefaults is a map of all Feature Gates that are
 	// operational in Agones, and what their default configuration is.
 	// alpha features are disabled.
 	featureDefaults = map[Feature]bool{
-		FeatureExample:               true,
-		FeaturePlayerTracking:        false,
-		FeatureSDKWatchSendOnExecute: true,
-		FeatureRollingUpdateOnReady:  true,
-		NodeExternalDNS:              false,
+		FeatureExample:                true,
```

Comment:
I wonder if we should keep these sorted somehow (alphabetical and/or based on true/false). Since they end up in a map the order they are inserted doesn't matter, but it may be useful to humans to have them in a logical ordering.

Reply:
🤔 they are currently in order of initial implementation (I think?).

I don't have strong opinions here tbh. Happy to take advice?